### ISSUE TITLE
installing asciio via pckage manager

### ISSUE THREAD

**Issue by nkh** [oguma/asciio-ditaa#1]
Hi, I think  it's possible to install asciio via package managers, which is much simpler, not for the perl part but for getting the glib and gtk part.

eg: https://tracker.debian.org/pkg/asciio

**Comment by oguma** [comment]
Agreed. I am not maintaining this repo, so I hope it goes upstream — see #2.
